SAFETY.md

```
@@ -0,0 +1,200 @@
+./scripts/quick-safe.sh install
+
+# Safe build operations
+./scripts/quick-safe.sh build
+
+# Safe file deletion
+./scripts/quick-safe.sh rm-rf
+```
+
+## Safety Metrics
+
+### Current Status
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)